_commands/basics/wc.md

```diff
@@ -2,23 +2,74 @@
 ---
 
 wc
--------
-TODO: Add documentation for this command by submitting a pull request.
-<!-- one line explanation would go here -->
+--
+
+`wc` prints newline, word, and byte counts for each FILE, and a total if more than one FILE is specified. With no FILE, or when FILE is a dash ("-"), wc operates on standard input. (A word is a non-zero-length sequence of characters delimited by white space.)
```

Comment:
A bit lengthy of a description for `wc`. Lots of good info here, but some could be put down below with other examples.

A shorter alternative for this could be:

> `wc` can print newline, word, and byte counts for each `FILE`.

Also, "FILE" can be put in backticks to render as code such as `FILE`.